### PR TITLE
Fixed: GetPowerShellModule installs package management modules into t…

### PR DESCRIPTION
**Pull request by splatteredbits**
…he custom install directory.

### CODE REVIEW

File: Whiskey/Whiskey.psd1 [hunk 1]
```diff
@@ -170,6 +170,7 @@
 * Removed all support for old "VersionFrom", "PrereleaseMap", and "Version" properties in the root of your whiskey.yml file. Use Whiskey's `Version` task instead.
 
 ## Improvements and Bug Fixes
+
```
Comment (DecoyJoe) [inline Whiskey/Whiskey.psd1]: Missing release notes

File: Whiskey/Tasks/GetPowerShellModule.ps1 [hunk 2]
```diff
@@ -50,7 +50,6 @@ function Get-WhiskeyPowerShellModule
                                               -Version $Version `
                                               -BuildRoot $TaskContext.BuildRoot `
                                               -AllowPrerelease:$AllowPrerelease `
-                                              -Path $fullPath `
```
Comment (DecoyJoe) [inline Whiskey/Tasks/GetPowerShellModule.ps1]: Are we removing the ability to specify a path for where the PowerShell modules will be downloaded to?

File: Whiskey/Whiskey.psd1 (hunk 1 above)
Comment (splatteredbits) [reply Whiskey/Whiskey.psd1]: This is a fix for new functionality that hasn't been released on master yet.

File: Whiskey/Tasks/GetPowerShellModule.ps1 (hunk 2 above)
Comment (splatteredbits) [reply Whiskey/Tasks/GetPowerShellModule.ps1]: No. We still use it when we install. We just don't need it when we find the version to install. Not sure why I ever passed the path to `Resolve-WhiskeyPowerShellModule`.

File: Whiskey/Whiskey.psd1 (hunk 1 above)
Comment (DecoyJoe) [reply Whiskey/Whiskey.psd1]: 👍

File: Whiskey/Tasks/GetPowerShellModule.ps1 (hunk 2 above)
Comment (DecoyJoe) [reply Whiskey/Tasks/GetPowerShellModule.ps1]: 👍

**Pull request merged**